docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,158 @@
+===========
+1.0 Alpha 3
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 3.
+
+
+Rust
```

Comment:
I'd call this section CLI

Reply:
I thought that "Rust" might highlight the nature of the change better, but we can make it CLI instead (and move the comment about internals rewritten in Rust to "Server")

Reply:
Well, we want the changelog to be succinct and to the point. The section is about CLI, and its rewrite to Rust is an important point, but not the only point.